src/compiler/diagnosticMessages.json

```diff
@@ -1696,7 +1696,7 @@
         "category": "Error",
         "code": 2473
     },
-    "In 'const' enum declarations member initializer must be constant expression.": {
+    "const enum member initializers can only contain literal values and other const enums values.": {

```

Comment:
Sorry, now that I look at it, this is somewhat inaccurate because any computed enum is fair game:

```ts
enum E {
    A,
    B,
    C,
}

const enum F {
    // works
    A = E.A,
}
```

What about

```
const enum member initializers can only contain literal values and other computed enum values.
```

Reply:
Sure, I will change the message.

BTW, wouldn't it make sense to provide some link to a page with samples and more detailed description in every error message? This error message already contains 2 Typescript terminologies (`const enum initializer`, `computed enum values`) which make it hard to understand. At least for me.

Reply:
I think down the line that might make sense, but I'd rather not do this for only one error message. I could imagine that down the line, diagnostics had an "documentation" field or something like that.

Reply:
Exactly. This was an idea for all the error messages. Should I create an issue?

Reply:
Sure thing! We've been discussing it recently.